pkg/rpcserver/rpcserver.go

```diff
@@ -49,12 +52,14 @@ type Server struct {
 	StatExecs      *stats.Val
 	StatNumFuzzing *stats.Val
 
-	cfg      *Config
-	mgr      Manager
-	serv     *flatrpc.Serv
-	target   *prog.Target
-	timeouts targets.Timeouts
-	checker  *vminfo.Checker
+	cfg          *Config
+	mgr          Manager
+	serv         *flatrpc.Serv
+	target       *prog.Target
+	timeouts     targets.Timeouts
+	checker      *vminfo.Checker
+	PCBase       uint64
```

Comment:
This should not be an exported field. But also we store whole Config in Server, so you can remove these new fields and just use `serv.cfg.PCBase` below.